phoenix-core-client/src/main/java/org/apache/phoenix/execute/MutationState.java

```diff
@@ -187,6 +187,9 @@ public class MutationState implements SQLCloseable {
 
     private final boolean indexRegionObserverEnabledAllTables;
 
+    private ReturnResult returnResult;
```

Comment:
Can you add some comments here explaining what this fields mean

Reply:
Sure

Reply:
Also, do we need to re-initialize them when the mutation state is `reset` ? If somebody re-uses the same connection .

Reply:
We can re-set it as part of `reset` but i think it is not strictly necessary because `PhoenixStatement` sets it only for auto-commit based new API usage, but its still good practice to reset it as part of the `MutationState` reset.